src/common/Paginator/index.js

```diff
@@ -31,52 +39,38 @@ const Paginator = () => {
         return () => window.removeEventListener('resize', handleResize);
     }, [mobileMax2]);
 
-    {/*const handleFirstPage = () => {
-        onPageChange(1);
-    };
-
-    const handlePreviousPage = () => {
-        if (currentPage > 1) {
-            onPageChange(currentPage - 1);
-        }
+    const handlePageChange = (newPage) => {
+        dispatch(setactualPage(newPage));
+        const basePath = location.pathname.includes("movies") ? "/movies" : "/people";
+        navigate(`${basePath}/page/${newPage}`);
     };
 
-    const handleNextPage = () => {
-        if (currentPage < totalPages) {
-            onPageChange(currentPage + 1);
-        }
-    };
-
-    const handleLastPage = () => {
-        onPageChange(totalPages);
-    };*/}
-
     return (
         <StyledPaginator>
             <BackwardForward>
-                <PaginatorButton> {/*onClick={handleFirstPage} disabled={currentPage === 1*/} 
+                <PaginatorButton onClick={() => handlePageChange(1)} disabled={actualPage === 1}>
                     <ButtonImage src={isMobile ? MobileFirstPageIcon : FirstPageIcon} alt="First page" />
                 </PaginatorButton>
-                <PaginatorButton> {/*onClick={handlePreviousPage} disabled={currentPage === 1}*/}
+                <PaginatorButton onClick={() => handlePageChange(actualPage - 1)} disabled={actualPage === 1}>
                     <ButtonImage src={isMobile ? MobilePreviousPageIcon : PreviousPageIcon} alt="Previous page" />
                 </PaginatorButton>
             </BackwardForward>
             <PageInfo>
                 <PageText>Page</PageText>
-                <PageNumber>1</PageNumber>
+                <PageNumber>{actualPage}</PageNumber>
                 <PageText>of</PageText>
                 <PageNumber>500</PageNumber>
             </PageInfo>
             <BackwardForward>
-                <PaginatorButton> {/*onClick={handleNextPage} disabled={currentPage === totalPages}*/}
+                <PaginatorButton onClick={() => handlePageChange(actualPage + 1)} disabled={actualPage === 500}>
                     <ButtonImage src={isMobile ? MobileNextPageIcon : NextPageIcon} alt="Next page" />
                 </PaginatorButton>
-                <PaginatorButton> {/*onClick={handleLastPage} disabled={currentPage === totalPages}*/}
+                <PaginatorButton onClick={() => handlePageChange(500)} disabled={actualPage === 500}>
```

Comment:
Will the value of the last page (500) be fixed, or is it entered temporarily and eventually generated dynamically? If it will be dynamic, care will have to be taken to handle the situation where the API returns a blank page.